Makefile

```diff
@@ -275,12 +268,20 @@ install-container: install-bpf
 	$(QUIET)$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
 	for i in $(SUBDIRS_CILIUM_CONTAINER); do $(MAKE) $(SUBMAKEOPTS) -C $$i install; done
 
+install-container-binary: install-bpf
+	$(QUIET)$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
+	for i in $(SUBDIRS_CILIUM_CONTAINER); do $(MAKE) $(SUBMAKEOPTS) -C $$i install-binary; done
+
+install-bash-completion:
+	$(QUIET)$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
+	for i in $(SUBDIRS_CILIUM_CONTAINER); do $(MAKE) $(SUBMAKEOPTS) -C $$i install-bash-completion; done
+
 # Workaround for not having git in the build environment
 # Touch the file only if needed
 GIT_VERSION: force
 	@if [ "$(GIT_VERSION)" != "`cat 2>/dev/null GIT_VERSION`" ] ; then echo "$(GIT_VERSION)" >GIT_VERSION; fi
 
-include Makefile.docker
+-include Makefile.docker
```

Comment:
Incidentally, relating to this specific line, within this patch series I saw the `-` at the start be removed then re-added between the two authors. What does this `-` achieve and do we really want it? (If it means "ignore failures", that doesn't sound like behaviour we want).

Reply:
cc @jrajahalme

Reply:
@joestringer I've added the patch for the microk8s as suggested. @jrajahalme can you explain on the `-` detail? Thanks

Reply:
`Makefile.docker` is now excluded from the docker build context so that when working with it locally docker build caches are not invalidated. `-` in the beginning allows the `include` to succeed even when the file does not exist. If the file is accidentally deleted the failure to find the named make target should be obvious.

Reply:
We have two sets of dockerignore files, `.dockerignore` along the `Dockerfile`s in `images` that are used by GH actions, and `Dockerfile.dockerignore` generated by `make` that uses dockerignore content autogenerated from the variaous `.gitignore` files in the repo, also excluding `.git` so that devs do not need to wait for 100's of MBs being transferred for each docker target.